scipy/stats/tests/test_morestats.py

```diff
@@ -735,33 +735,35 @@ def test_alternative_approx(self):
         assert_allclose(pval_l, 1-pval/2, atol=1e-12)
 
 
+@array_api_compatible
```

Comment:
```suggestion
@array_api_compatible
@pytest.mark.usefixtures("skip_xp_backends")
@pytest.mark.skip_xp_backends(cpu_only=True, reasons=['Uses NumPy for pvalue'])
```
This skips the test for CuPy, which isn't running on CI (so the failure is not visible).